lerobot-web-fe/src/services/teleoperate.service.ts

```diff
@@ -1,10 +1,13 @@
+import { useStatusStore } from '../stores/status.store';
 import type { RobotIds, TeleoperateResponse } from '../models/teleoperate.model';
 
-const API_BASE = 'http://127.0.0.1:80';
-
 export async function startTeleoperate({ leader, follower }: RobotIds): Promise<TeleoperateResponse> {
+  const { apiUrl } = useStatusStore.getState();
```

Comment:
Instead of destruct, please use the separated selector-based solution here in this file as well.

```suggestion
  const apiUrl = useStatusStore((s) => s.apiUrl);
```

Reply:
It will cause an invalid hook call. The useStatusStore(s => s.apiUrl) is for reactive, efficient updates. The  useStatusStore.getState().apiUrl usage is safe, static, and meets React's hook rules.